---
layout: page
title: "Skills"
permalink: /skills
---

## Statistical genetics and omics analysis: 
Mendelian randomization, statistical fine-mapping, colocalization, differential expression analysis using bulk RNA-sequencing data, weighted gene co-expression network analysis (WGCNA), similarity network fusion (SNF), pathway enrichment analysis, gene-by-environment interactions.
Hands-on experience in designing and executing data quality control pipelines for epigenome-wide DNA methylation (Illumina array), proteomics (Olink platform), metabolomics (mass spectrometry), and genotyping array data.  

## Longitudinal and survival data analysis:
Mixed effects models, generalized estimating equations (GEE), latent growth mixture modelling, survival analysis (non-parametric, semi-parametric, parametric), mediation analysis.

## Programming and computation:
R, Python (NumPy, SciPy, Matplotlib, Pandas, scikit-learn), SAS, Linux and shell scripting, experience in working in the high-performance computing environment (e.g., SLURM, PBS) including executing parallel computing jobs through R package `doParallel`.

## Coursework for biology and content knowledge:
Concepts of Molecular Biology, Principles of Immunology, Epidemiology of Aging, Epidemiology of Cardiovascular Disease, Introduction to Clinical Trials, Molecular Epidemiology and Biomarkers in Public Health

## Coursework for biotech/pharma commercialization:
Duke University Regulatory Affairs Training Program (Fall 2023), Morrison Foerster Patent Law Course (Fall 2023)

## Language and other skills:
Chinese (mandarin, native), English (fluent), excellent written and verbal communication, tennis (USTA NTRP 4.0)
